### PR TITLE
Content page styling

### PR DESCRIPTION
**Pull request by whitzhu**
screen shot 2016-08-02 at 3 36 33 pm
## Summary

Moved title & description below content render.

*Short description*

## TODO

- [ ] Have tests been written for the new code?
- [ ] Has documentation been written/updated?
- [ ] New dependencies (if any) added to requirements file
- [ ] Add an entry to CHANGELOG.rst
- [ ] Add yourself it AUTHORS.rst if you don't appear there

## Reviewer guidance

*If you PR has a significant size, give the reviewer some helpful remarks*

## Issues addressed

List the issues solved or partly solved by the PR

## Documentation

If the PR has documentation, link the file here (either .rst in your repo or if built on Read The Docs)

## Screenshots (if appropriate)

They're nice. :)

### CODE REVIEW

File: kolibri/plugins/learn/assets/src/vue/content-page/index.vue [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
   <div>
 
-    <page-header :title='title'>
+    <page-header>
```
Comment (indirectlylit) [inline kolibri/plugins/learn/assets/src/vue/content-page/index.vue:5]: looks like you have two `page-header` tags - probably not intended?

File: kolibri/plugins/learn/assets/src/vue/content-page/index.vue [hunk 2]
```diff
@@ -103,6 +106,8 @@
 
 <style lang="stylus" scoped>
 
+  @require '~core-theme.styl'
+
```
Comment (indirectlylit) [inline kolibri/plugins/learn/assets/src/vue/content-page/index.vue:110]: this is added but looks unused?

**Comment by indirectlylit** [comment]
do these changes work on all content types?

**Comment by whitzhu** [comment]
@indirectlylit yea, this moves the title & description of all content-type-page below the video render.

**Pull request merged**